src/components/tabset/tabset.component.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { TabsetModule } from './tabset.module';
+import { Component } from '@angular/core';
+import { dispatchMouseEvent } from '../../common/testing';
+
+@Component({
+    selector: 'app-radio-button-value-test',
+    template: `
+        <ux-tabset [minimal]="true" aria-label="Tabset Example">
+            <ux-tab (deselect)="onTabDeselect()"  (deactivated)="onTabDeactivated()" (select)="onTabSelectClick()"  (activated)="onTabActivatedClick()" customClass="text-center" *ngFor="let tab of tabs">
+                <ng-template uxTabHeading>
+                    <span>{{ tab.title }}</span>
+                </ng-template>
+
+                <div class="m-t">
+                    <h4>{{ tab.title }}</h4>
+                    <p>{{ tab.content }}</p>
+                </div>
+            </ux-tab>
+        </ux-tabset>
+    `
+})
+export class TabsetTestComponent {
+
+    tabs: Tab[] = [
+        {
+            content: 'test text',
+            title: 'Schedule',
+        },
+        {
+            content: 'test text',
+            title: 'Protection',
+        },
+        {
+            content: 'test text',
+            title: 'Solution',
+        },
+        {
+            content: 'test text',
+            title: 'Analytics',
+        }
+    ];
+
+    onTabSelectClick(): void {
```

Comment:
Could we remove the `Click` from the function names

Reply:
removed